config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -0,0 +1,74 @@
+name: pangeo-hubs
+provider: gcp
+gcp:
+  key: secrets/pangeo-hubs.json
+  project: pangeo-integration-te-3eea
+  cluster: pangeo-hubs-cluster
+  zone: us-central1-b
+support:
+  nfs-server-provisioner:
+    enabled: true
+hubs:
+  - name: staging
+    domain: staging.pangeo.2i2c.cloud
+    template: daskhub
+    auth0:
+      connection: github
+    config: &stagingConfig
+      basehub:
+        nfsPVC:
+          enabled: false
+        jupyterhub:
+          proxy:
+            https:
+              enabled: false
+          custom:
+            homepage:
+              templateVars:
+                org:
+                  name: Pangeo
+                  url: https://pangeo.io
+                  logo_url: "https://coessing.files.wordpress.com/2016/08/ghana-logo-21.png?w=262&h=376&zoom=2"
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name: Pangeo
+                  url: https://pangeo.io
```

Comment:
@rabernat Is this correct reference for funding source or should it be Columbia and/or something else?

Reply:
Pangeo is not really a funding agency. We _receive_ funding for various Pangeo-related projects from sponsors.

The current funding source is The Gordon and Betty Moore Foundation (https://www.moore.org/), so that's probably what should go here for now.

Going forward, we will soon switch to NSF Earthcube funding.